libs/@local/hash-graph-types/graph_types/_const.py

```diff
@@ -0,0 +1,70 @@
+from typing import Annotated, Any, TypeVar, cast, overload, Literal
```

Comment:
For what `Literal` is used here?

Reply:
fixed in [8c97d8b](https://github.com/hashintel/hash/pull/2776/commits/8c97d8b9711344e7b703a28c2083e6ceeb4c66ad)